src/dismissible.css

```diff
@@ -110,4 +111,48 @@
   100% {
     transform: translateY(0);
   }
+}
+
+/*--------------------------------
```

Comment:
Do you think that maybe we include line icon classes in a separate CSS file? Ideally the library won't require a specific icon set to be used, but this can be included as an option.

Reply:
I think this is a good idea.
I've made a commit with changes to separate Line Icons.
I will create a way to import or not Line Icons and I will create a merge request.
It's okay for you?

Reply:
@brookesb91 , I've updated the README file with information of how to replace default icons.
What do you think?

Reply:
Looks good to me!
I will need to fix the release action before we merge, however!